test/fasta.py

```diff
@@ -3,7 +3,7 @@
 from dark.fasta import FastaReads
 
 
-@lru_cache
+@lru_cache()
```

Comment:
How come these are added? I'm not sure that's legit.....

Reply:
Otherwise errors happen when running this code in Python <3.8

Reply:
Huh, ok. I just tried with 3.7 and see it too...